src/index.js

```diff
@@ -55,6 +55,7 @@ GoogleFontLoader.propTypes = {
     ).isRequired,
     subsets: PropTypes.arrayOf(PropTypes.string),
     display: PropTypes.string,
+    contentDocument: PropTypes.node,
```

Comment:
Is this the right prop type for this? `PropTypes.node` is something that can be rendered but the default `document` doesn't fit that, I think?

Reply:
eep. Yeah, you're right. I'm not sure what the right `PropType` for `document` would be. I'll do some research.